cpg-language-llvm/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/llvm/LLVMIRLanguageFrontendTest.kt

```diff
@@ -991,12 +991,11 @@ class LLVMIRLanguageFrontendTest {
 
         val innerCatchThrows =
             (innerTry.catchClauses[0].body?.statements?.get(1) as? IfStatement)?.elseStatement
-                as? UnaryOperator
-        assertNotNull(innerCatchThrows)
-        assertNotNull(innerCatchThrows.input)
+        assertIs<ThrowStatement>(innerCatchThrows)
+        assertNotNull(innerCatchThrows.exception)
         assertSame(
```

Comment:
should use `assertRefersTo` here

Reply:
I'd update the remaining tests in a separate PR. This one is halfway cleaned up but still ugly.